onnxruntime/core/providers/stvm/stvm_allocator.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "stvm_allocator.h"
+#include "core/framework/allocatormgr.h"
+#include "core/framework/session_state.h"
+#include "xpu_data_transfer.h"
+
+
+namespace onnxruntime {
+
+
+void* STVMAllocator::Alloc(size_t size) {
+  //LOG(INFO) << "allocating " << size;
```

Comment:
Remove these dead LOG lines or switch to DLOG?

Reply:
I've removed dead code, comments and logs here and below